public/js/hits.js

```diff
@@ -210,27 +210,27 @@ class Hits extends Component {
         // Highlight selected hit and enable 'Download FASTA/Alignment of
         // selected' links.
         if (checkbox.is(':checked')) {
-            $hit.addClass('glow');
-            $hit.next('.hsp').addClass('glow');
+            $hit.addClass('border-l-2 border-seqorange pl-1');
+            $hit.next('.hsp').addClass('border-l-2 border-seqorange pl-1');
```

Comment:
I like the highlight, but it also shifts both blocks to the right. I think it would be nice to offset the added border by a negative left margin or something so that checking the box does not move elements visually

Reply:
updated.